webui/react/src/pages/ExperimentDetails/ExperimentTrials.tsx

```diff
@@ -398,14 +402,15 @@ const ExperimentTrials: React.FC<Props> = ({ experiment }: Props) => {
   return (
     <>
       <Section>
-        <TableBatch selectedRowCount={selectedRowKeys.length} onClear={clearSelected}>
-          <Button onClick={(): Promise<void> => handleBatchAction(Action.OpenTensorBoard)}>
-            View in TensorBoard
-          </Button>
-          <Button onClick={(): Promise<void> => handleBatchAction(Action.CompareTrials)}>
-            Compare Trials
-          </Button>
-        </TableBatch>
+        <TableBatch
+          actions={[
+            { label: Action.OpenTensorBoard, value: Action.OpenTensorBoard },
+            { label: Action.CompareTrials, value: Action.CompareTrials },
+          ]}
```

Comment:
I like this way of defining actions much better than manually having buttons as the children!